### DISCUSSION TITLE
Stanford 计算机科学专业 | HackWay技术学习路线

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#36] [Announcements]
# Stanford 计算机科学专业 | HackWay技术学习路线

说到硅谷则必提及斯坦福，斯坦福大学为硅谷的形成与发展奠定了坚实的基础，培养了众多高科技公司、创始人（如惠普、雅虎、谷歌、WhatsApp、Netflix等）与优秀工程师，同时也是图灵奖得主最多的高校。斯坦福大学的课程不仅紧随一线企业的发展需求，还起到引领作用，国内高校的课程则往往落后企业发展十数年之久。

https://hackway.org/docs/cs/path/stanford

**Comment by BriefAnswer** [comment]
关于课号的介绍有问题，在 https://www.topsedu.com/about/news/20546.html 找到了比较合理的介绍：计算机系的课号是CS0到CS599，其中：0~99号是服务性课程，适合非技术性专业的学生选择;100~199号是本科生基础课程;200~299号是高级本科生课程/初级研究生课程;300~399号是高级研究生课程;后边的课号代表实验和讨论班。

**Reply by BriefAnswer** [reply]
评论错地方了，显示也出问题了，忽略忽略……
